build/release/release.sh

```diff
@@ -16,6 +16,35 @@ if [[ $1x = x ]]; then
     TAG=$DATE-$VERSION
 else
     TAG=$1
+
+    # If a tag is specified, ensure that that tag is present and checked out.
+    if [[ $TAG != $(git tag -l --points-at HEAD) ]]; then
+        echo "tag $TAG not checked out"
+        exit 1
+    fi
+
+    # Build lnd to extract version.
+    go build github.com/lightningnetwork/lnd/cmd/lnd
+
+    # Extract version command output.
+    LND_VERSION_OUTPUT=`./lnd --version`
+
+    # Use a regex to isolate the version string.
+    LND_VERSION_REGEX="lnd version (.+) commit"
+    if [[ $LND_VERSION_OUTPUT =~ $LND_VERSION_REGEX ]]; then
+        # Prepend 'v' to match git tag naming scheme.
+        LND_VERSION="v${BASH_REMATCH[1]}"
```

Comment:
So the tags _do_ have that prefix, but the formatted version output for the `lnd/lncli` commands _don't_: 
```
lnd --version
lnd version 0.8.2-beta commit=queue/v1.0.2-165-g4273bc0ba22550536625c73d54539a7f6bc6030e
```

Reply:
What is the requested change? The comment says "prepend 'v' (to parsed version) to match git tag naming". I think that is correct.